src/app/shared/services/auth.service.ts

```diff
@@ -0,0 +1,44 @@
+import { userObject } from './../interfaces/userObject';
+import { MessageService } from '@shared/services/message.service';
+import { Injectable } from '@angular/core';
+import { LoginObject } from '@shared/interfaces/loginObj';
+import { Observable, BehaviorSubject, of } from 'rxjs';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+// import { FormingApiUrlService } from './auxiliaryServices/forming-api-url.service';
+// import { catchError } from 'rxjs/operators';
+
+const httpOptions = {
+  headers: new HttpHeaders({ 'Content-Type': 'application/json' })
```

Comment:
I think we discussed this. It can be part of the interceptor, but in your case, it is needed only for the methods here, so we are okay.